inst/extremes/extremes.Rmd

```diff
@@ -26,5 +26,8 @@ cat(getLogo())
 tbl <- extremesTable(data)
 # formTable <- padTable(tbl)
 # cat(formTable)
-kable(tbl, align='r', row.names=FALSE)
+if(!identical(tbl, "The dataset requested is empty.")){
```

Comment:
Yes, there are a lot of instances where we need to consolidate shared logic, that's why utils-shared was created.